doc/description.md

```diff
@@ -0,0 +1,132 @@
+# Description
+
+Audit records are created locally on each Cassandra node covering all CQL requests.
+ecAudit has a pluggable logger backend with support for SLF4J and Chronicle Queues out of the box.
+The SLF4J logging framework makes it possible to store and format audit logs in various ways.
+The binary Chronicle Queues offers the best possible performance.
+It is possible to create a whitelist of usernames/resources which will not be included in the audit log.
+
+ecAudit will create an audit record before the operation is performed, marked with status=ATTEMPT.
+If the operation fails, for one reason or another, a corresponding failure record will be created, marked with status=FAILED.
+
+Passwords which appear in an audit record will be obfuscated.
+
+
+## Audit Records
+
+| Field Label | Field Value                                                       | Mandatory Field |
+| ----------- | ----------------------------------------------------------------- | --------------- |
+| client      | Client IP address                                                 | yes             |
+| user        | Username of the authenticated user                                | yes             |
+| batchId     | Internal identifier shared by all statements in a batch operation | no              |
+| status      | Value is either ATTEMPT or FAILED                                 | yes             |
+| operation   | The CQL statement or a textual description of the operation       | yes             |
```

Comment:
Should TIMESTAMP be described here as well? (was not before) but it might be a bit confusing to have 2 tables with "almost" the same data...
It exists in doc/slf4j_logger.md, but it is part of the Chronicle Logger as well

Reply:
Fixed!